schunk_powercube_chain/ros/src/schunk_powercube_chain.cpp

```diff
@@ -374,7 +374,10 @@ class PowerCubeChainNode
     std::vector<double> Offsets(DOF);
     for (unsigned int i = 0; i < DOF; i++)
     {
-      Offsets[i] = model.getJoint(JointNames[i].c_str())->calibration->rising.get()[0];
+      if(model.getJoint(JointNames[i].c_str())->calibration == 0)
```

Comment:
Should it rather be `if(model.getJoint(JointNames[i].c_str())->calibration == NULL)` as `calibration` is a pointer?